src/lib/linux-api/src/time.rs

```diff
@@ -0,0 +1,68 @@
+use bytemuck::TransparentWrapper;
+use core::result::Result;
+use linux_syscall::syscall;
+use linux_syscall::Result as LinuxSyscallResult;
+use num_enum::{IntoPrimitive, TryFromPrimitive};
+
+use crate::bindings;
+use crate::errno::Errno;
+
+pub use bindings::linux___kernel_clockid_t;
+
+// Clocks
+#[derive(Debug, Copy, Clone, Eq, PartialEq, IntoPrimitive, TryFromPrimitive)]
+#[repr(i32)]
+#[allow(non_camel_case_types)]
+pub enum ClockId {
+    CLOCK_REALTIME = bindings::LINUX_CLOCK_REALTIME as linux___kernel_clockid_t,
+    CLOCK_MONOTONIC = bindings::LINUX_CLOCK_MONOTONIC as linux___kernel_clockid_t,
+    CLOCK_PROCESS_CPUTIME_ID = bindings::LINUX_CLOCK_PROCESS_CPUTIME_ID as linux___kernel_clockid_t,
```

Comment:
Since we're casting the Linux types in most of our structs/enums, maybe we should have static assertions for them to make sure we're not truncating the values?